docs/reference/notices.md

```diff
@@ -1,13 +1,42 @@
 ---
 id: notices
 title: "Security notices"
-description: "Let's take a closer look at security notices, reporting vulnerabilities, and addiitonal security information."
+description: "Let's take a closer look at security notices, reporting vulnerabilities, and additional security information."
 ---
 
 ## Security notices
 
 Camunda publishes security notices after fixes are available.
 
+### Notice 14
+
+#### Publication date
+
+March 11th, 2025
+
+#### Product affected
+
+Camunda Web Modeler
+
+#### Impact
+
+The version of `koa` used by Camunda Web Modeler was affected by the following vulnerability:
+
+- https://nvd.nist.gov/vuln/detail/CVE-2025-25200
+
+#### How to determine if the installation is affected
+
+You are using Camunda Web Modeler version ≤ 8.3.16, ≤ 8.4.14, ≤ 8.5.15, or ≤ 8.6.7.
+
+#### Solution
+
+Camunda has provided the following releases which contain the fix:
+
+- [Camunda Web Modeler 8.3.17](https://github.com/camunda/web-modeler/releases/tag/self-managed%2F8.3.17)
+- [Camunda Web Modeler 8.4.15](https://github.com/camunda/web-modeler/releases/tag/self-managed%2F8.4.15)
+- [Camunda Web Modeler 8.5.16](https://github.com/camunda/web-modeler/releases/tag/self-managed%2F8.5.16)
+- [Camunda Web Modeler 8.6.8](https://hub.docker.com/layers/camunda/web-modeler-webapp/8.6.8/images/sha256-0d12ad2a751d9cc78de25f7a299320c8b7f7fa68f3b2e80f0c826fee926cb547)
```

Comment:
🔧 I don't think linking the webapp image provides any additional value here - my proposal would be to just completely remove the links and only mention the versions.

Reply:
I removed the links